src/components/NFTPortfolio/NonWritingItem.vue

```diff
@@ -0,0 +1,96 @@
+<template>
```

Comment:
What is the significant difference between non-writing items& writing nft item? Seem very similar in layout, could be refactored

Reply:
Refactored